portal-api/src/v1/controllers/amendments/put-amendment.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import { ApiError, CustomExpressRequest, PortalFacilityAmendmentUserValues } from '@ukef/dtfs2-common';
+import { generatePortalAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import api from '../../api';
+
+export type PutAmendmentRequest = CustomExpressRequest<{
+  params: {
+    facilityId: string;
+  };
+  reqBody: PortalFacilityAmendmentUserValues;
```

Comment:
Shouldn't cause any issues here but unless we convert the body using zod, this will be slightly disingenuous (dates will be strings)